### PR TITLE
Cyber ark xsiam Demo fixes

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Some demo and modelling rule fixes

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by dantavori — APPROVED** [review]
make sure to merge after verifying that when you upload the pack from marketplace, the modeling rules appear in the UI.

**Review by yaakovi — APPROVED** [review]

File: Packs/CyberArkIdentity/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+This pack includes XSIAM content.
+
```
Comment (bakatzir) [inline Packs/CyberArkIdentity/README.md:2]: why do we need this sentence?

Comment (MosheEichler) [reply Packs/CyberArkIdentity/README.md]: Its a part of the pack README convention [here](https://docs.google.com/document/d/16CKdwnQF9r0IZPSZUM34NWFJHX1IEcdAe0vnvIrqZ2w/edit)

@tomer-pan @meital-aizen @tkatzir FYI

Comment (bakatzir) [reply Packs/CyberArkIdentity/README.md]: It make sense when the pack it deployed to MP v1 as well, but this is not the case here.

**Pull request merged**